submissions/sprint_b/app.py

```diff
@@ -0,0 +1,29 @@
+from . import app
+from flask import jsonify
+from .helpers import EpithetGenerator, Vocabulary
+
+
+@app.route("/")
+def generate_epithets():
+    epithet_generator = EpithetGenerator()
+    final_string = epithet_generator.generate_epithets()
+
+    return jsonify({"epithets": final_string})
+
+@app.route("/<int:insults>")
+def insults(insults):
+    epithet_generator = EpithetGenerator()
+    insult_list = []
+    i = 0
+    while i < insults:
```

Comment:
Your generate_epithets method should accept the quantity of epithets to generate and return a list of epithets. All of the logic for looping some quantity of times should be part of your generate_epithets method.